test/acvp_kyber.c

```diff
@@ -0,0 +1,269 @@
+// SPDX-License-Identifier: Apache-2.0
+#include <stddef.h>
+#include <stdio.h>
+#include <string.h>
+#include "kem.h"
+#include "randombytes.h"
+
+#define USAGE \
+  "acvp_kyber{lvl} [encapDecap|keyGen] [AFT|VAL] {test specific arguments}"
+#define ENCAPS_USAGE "acvp_kyber{lvl} encapDecap AFT encaps ek=HEX m=HEX"
+#define DECAPS_USAGE "acvp_kyber{lvl} encapDecap VAL decaps dk=HEX c=HEX"
+#define KEYGEN_USAGE "acvp_kyber{lvl} keyGen AFT z=HEX d=HEX"
```

Comment:
Not sure if it is just me, but this argument list suggests to me that I can change the order of the arguments and it will still work. But maybe it's still a bit better than just relying on the order.

Reply:
No, it's ordered -- I think it's good enough for our purposes.